src/lib/features/feature-search/feature-search-store.ts

```diff
@@ -264,13 +245,48 @@ class FeatureSearchStore implements IFeatureSearchStore {
                 'total_features',
                 this.db.raw('select count(*) as total from final_ranks'),
             )
+            .with('metrics', (queryBuilder) => {
+                queryBuilder
+                    .sum('yes as yes')
+                    .sum('no as no')
+                    .select([
+                        'client_metrics_env.environment',
+                        'client_metrics_env.feature_name',
+                    ])
+                    .from('client_metrics_env')
+                    .innerJoin(
+                        'final_ranks',
+                        'client_metrics_env.feature_name',
+                        'final_ranks.feature_name',
+                    )
+                    .where(
+                        'client_metrics_env.timestamp',
+                        '>=',
+                        this.db.raw("NOW() - INTERVAL '1 hour'"),
+                    )
+                    .groupBy([
+                        'client_metrics_env.feature_name',
+                        'client_metrics_env.environment',
+                    ]);
+            })
             .select('*')
             .from('ranked_features')
             .innerJoin(
                 'final_ranks',
                 'ranked_features.feature_name',
                 'final_ranks.feature_name',
             )
+            .leftJoin('metrics', (qb) => {
```

Comment:
Join it later, just to get the data.

Reply:
I think this change might have introduced a bug -- when you join here, fields of `metrics` are also in the final result, duplicating the column names `feature_name` and `environment`. When there are no metrics, the fields can be `null` here (leftJoin, after all), causing problems with the aggregation of the results.

https://github.com/Unleash/unleash/issues/6988